docs/project/breaking-change-process.md

```diff
@@ -0,0 +1,44 @@
+# Breaking change process
```

Comment:
Not all breaking changes are created equal.  Removing support for some feature is likely much more impactful than, say, switching the order of two argument validation exceptions that might occur if multiple invalid inputs are passed to a method.  Practically any bug fix can break someone.

What is the bar used for this process to define "breaking change"?

Reply:
I have a few thoughts on this:

- It's hard to define a definite bar. 
- The breaking change docs that I linked to were the ones that were in the repo, mostly from the old corefx repo, I believe. We can file issues if we think they are inadequate.
- In terms of determination, I think of it as a layered approach. People should make a first determination in the early design phase. It would be late, but the person pressing the "big green button" needs to make a determination of the information available on whether a change is breaking. The last resort is .NET users coming to this conclusion in preview or stable releases.

In short, this document doesn't define that, and I think that's fine.